tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/dav/DavClientException.java

```diff
@@ -19,12 +19,27 @@
 package com.linagora.tmail.dav;
 
 public class DavClientException extends RuntimeException {
+    private final boolean shouldRetry;
```

Comment:
I would rather subclass with a `RetriableDavClientException` rather than introduce an evil boolean parameter.

The semantic is also better: rather than saying "retry meeee" the semantic would then explain that this could be retried.

A simple type check would also be enough and the cast avoided altogether...